trades.py

```diff
@@ -622,6 +622,7 @@ def parse_user(  # pylint: disable=too-many-locals,too-many-statements
 ) -> tuple[str, dict[Any, Any]]:
     """Parse the trades from the response."""
 
+    _mech_statistics = dict(mech_statistics)
```

Comment:
Is the function's type hint correct?

Reply:
Fixed type `dict` to `Dict` globally.

Reply:
Not what I meant :sweat_smile: 
`dict` was fine. 
So you are just copying I assume. Why are you copying?

Reply:
I suspect that you can avoid copying by not shadowing `mech_statistics` from the `main` scope.

Reply:
I need both the original read-only parameter (which within the method I don't care if it shadows the main one) and I also need a copy of it which I can modify. Both are required in this method: the original and the modifiable one.